hyper/tls.py

```diff
@@ -29,14 +29,16 @@ def wrap_socket(sock, server_hostname, ssl_context=None, force_proto=None):
     A vastly simplified SSL wrapping function. We'll probably extend this to
     do more things later.
     """
-    global _context
+    if ssl_context:
+        # if an SSLContext is provided then use it instead of default context
+        _ssl_context = ssl_context
+    else:
+        global _context
```

Comment:
I...don't think you can do this. `global _context`, however, is just a statement that makes it clear to Python that I want to be able to assign to the global variable, so it shouldn't be needed.